shell-definition.yaml

```diff
@@ -13,7 +13,7 @@ imports:
   - cloudshell_standard: cloudshell_cloud_provider_standard_1_0_0.yaml
 
 node_types:
-  vendor.resource.Openstack Shell 2G:
+  vendor.resource.OSF-OpenStack-Cloud-Provider-Shell-2G:
```

Comment:
This is what the user would see on the Inventory tab on the portal, I believe we need to remove `-`